### DISCUSSION TITLE
Error: Could not find CLIPVision model model.safetensors for SD1.5.

### DISCUSSION THREAD

**Discussion by scottonanski** [Acly/krita-ai-diffusion#502] [General]
Getting this error but no useful information on where to acquire the need file.

Any help?

![image](https://github.com/Acly/krita-ai-diffusion/assets/110937649/dd23998a-d7ab-4fa7-8439-1cd741e626d5)

**Comment by Acly** [comment]
https://github.com/Acly/krita-ai-diffusion/wiki/ComfyUI-Setup

This lists all required models, info about how to download optional models, and troubleshooting hints.
